src/ssz/type/bit_list.zig

```diff
@@ -248,7 +248,7 @@ pub fn BitListType(comptime _limit: comptime_int) type {
         pub const Element: type = BoolType();
         pub const limit: usize = _limit;
         pub const Type: type = BitList(limit);
-        pub const TreeView: type = ListBasicTreeView(@This());
```

Comment:
@GrapeBaBa  why does the `ListBasicTreeView` not work? they are all about getting a node, and extract to a `bool` from an index

Reply:
ah I see. So it's the issue of `itemsPerChunk` not the ListBasicTreeView itself.
we should fix `itemsPerChunk()` instead?